src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -33,7 +33,7 @@ azure-mgmt-consumption==2.0.0
 azure-mgmt-containerinstance==9.1.0
 azure-mgmt-containerregistry==8.2.0
 azure-mgmt-containerservice==19.1.0
-azure-mgmt-core==1.3.0
+azure-mgmt-core==1.3.1
```

Comment:
Any specific reason this had to change?

Reply:
In package `azure-mgmt-rdbms==10.2.0b1` we have the requirement: `Requires-Dist: azure-mgmt-core (<2.0.0,>=1.3.1)`